packages/react-magma-dom/src/components/Select/Select.tsx

```diff
@@ -48,6 +48,7 @@ export function Select<T>(props: SelectProps<T>) {
     selectedItem: passedInSelectedItem,
     setReference,
     setFloating,
+    initialHighlightedIndex,
```

Comment:
This is already a prop, but it's not documented. Can we properly add it to the props list with an explanation of when it's needed?